tests/mlflow/tensorflow/test_tensorflow_model_export.py

```diff
@@ -55,32 +77,108 @@ def test_log_saved_model(self):
             try:
                 # Creating dict of features names (str) to placeholders (tensors)
                 feature_spec = {}
-                for name in self._feature_names:
+                for name in feature_names:
                     feature_spec[name] = tf.placeholder("float", name=name, shape=[150])
-                # Creating receiver function for model saving.
-                receiver_fn = tf.estimator.export.build_raw_serving_input_receiver_fn(feature_spec)
-                saved_model_path = tmp.path("model")
-                os.makedirs(saved_model_path)
-                os.makedirs(tmp.path("hello"))
-                # Saving Tensorflow model.
-                saved_model_path = self._dnn.export_savedmodel(saved_model_path, 
-                                                               receiver_fn).decode("utf-8")
-                # Logging the Tensorflow model just saved.
-                tensorflow.log_saved_model(saved_model_dir=saved_model_path,
-                                           signature_def_key="predict", 
-                                           artifact_path=tmp.path("hello"))
-                # Loading the saved Tensorflow model as a pyfunc.
-                x = pyfunc.load_pyfunc(saved_model_path)
-                # Predicting on the iris dataset using the pyfunc.
-                xpred = x.predict(pandas.DataFrame(data=self._X, columns=self._feature_names))
+
+                saved = []
+                for s in dnn_predict:
+                    saved.append(s['predictions'])
+
+                results = self.helper(feature_spec, tmp, dnn, pandas.DataFrame(data=X, columns=feature_names))
+
+                # Asserting that the loaded model predictions are as expected.
+                np.testing.assert_array_equal(saved, results)
+            finally:
+                # Restoring the old logging location.
+                tracking.end_run()
+                tracking.set_tracking_uri(old_tracking_dir)
+
+
+    def test_cat_cols(self):
```

Comment:
I'd call this test_categorical_columns.. no need to be that concise in a test case name